autograder/submitty_autograding_shipper.py

```diff
@@ -948,6 +960,40 @@ def checkout_vcs_repo(config, my_file):
                 #    # and check out the right version
                 #    subprocess.call(['git', 'checkout', '-b', 'grade', what_version])
 
+                # copy the subdirectory we want to the
+                # original checkout path and remove the extra files
+                if subdirectory_grading:
```

Comment:
as noted above, make this not a special case, but do it for all vcs gradeables, even with empty string subdirectory.